PWGHF/Core/SelectorCuts.h

```diff
@@ -303,6 +303,55 @@ static const std::vector<std::string> labelsPt = {
 static const std::vector<std::string> labelsCutVar = {"m", "DCA", "cos theta*", "pT K", "pT Pi", "d0K", "d0pi", "d0d0", "cos pointing angle", "cos pointing angle xy", "normalized decay length XY", "decay length", "decay length XY", "minimum decay length"};
 } // namespace hf_cuts_d0_to_pi_k
 
+namespace hf_cuts_dstar_to_pi_d0
+{
+static constexpr int nBinsPt = 10;
+static constexpr int nCutVars = 10;
+// default values for the pT bin edges (can be used to configure histogram axis)
+// offset by 1 from the bin numbers in cuts array
+constexpr double binsPt[nBinsPt + 1] = {
+  1.0,
+  1.5,
+  2.0,
+  2.5,
+  3.0,
+  3.5,
+  4.0,
+  4.5,
+  5.0,
+  5.5,
+  6.0};
+auto vecBinsPt = std::vector<double>{binsPt, binsPt + nBinsPt + 1};
+
+// row labels
+static const std::vector<std::string> labelsPt = {
+  "pT bin 0",
+  "pT bin 1",
+  "pT bin 2",
+  "pT bin 3",
+  "pT bin 4",
+  "pT bin 5",
+  "pT bin 6",
+  "pT bin 7",
+  "pT bin 8",
+  "pT bin 9"};
+
+// column label
+static const std::vector<std::string> labelsCutVar = {"minpTSoftPi", "maxpTSoftPi", "minpTD0_Prong0", "minpTD0_Prong1", "d0Softpi", "d0SoftPiNormalised", "DeltaMDstar", "Chi2PCA", "d0NormalisedProng0", "d0NormalisedProng1"};
```

Comment:
Fix capitalisation

Reply:
Capitalization in the cut variable? What are the guidelines for them? I got this [ALICE CODING GUIDELINES](https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names)

Reply:
Yes, the cut variables. Please follow the naming conventions for variables.